### PR TITLE
The Independent Suit Update

### PR DESCRIPTION
**Pull request by Apogee-dev**
## About The Pull Request
mixed_spread
Adds several independent space suits, makes some minor adjustments to independent-themed clothing, and adds the new clothes to the Rigger.

Space suit details:
- Independent security hardsuit. An outdated, armored hardsuit, several years older than Nanotrasen's security suits. This suit has higher bullet protection at the expense of laser protection. Total armor value is identical to the NT sec hardsuit.
- Independent mining hardsuit. A civilian-market, somewhat outdated alternative to Nanotrasen's mining hardsuits. Like the sec suit, has slightly more bullet armor than laser armor.
- Independent engineering space suit. A "soft" suit with enhanced thermal and radiation shielding and no other armor to speak of. The helmet, with its integrated hard hat, also has protection against blunt impacts and a built-in headlight.
- Pilot space suit. A very old space suit design that makes boarding and ejecting from mechs and space pods twice as fast. There are eight different skins for the helmet, with a random one chosen upon spawn! While it has extra built-in storage and is useful for vehicle operators, it is still completely unarmored.

Other changes:
- Hazard jumpsuits for engineers and miners now roll down when toggled instead of just rolling up the sleeves.
- Spacepods have a small eject delay identical to mechs.
- the Rigger has been updated with the new space suits.
- Rigger's wrecked Ripley has been replaced with a fully functioning Ripley! (some assembly may be required)

## Why It's Good For The Game

Faction drip and swanky new suits, what's not to love?

## Changelog
:cl:
add: Added four new Independent-themed space suits
tweak: Replaced Nanotrasen hard suits in cargo and vendors with their generic equivalents
tweak: Updated Rigger with new space suits
tweak: Rigger command and armory lockers are now locked by default
tweak: Replaced wrecked Ripley on the Rigger with an unassembled Ripley
tweak: Space pods now have a small eject delay for consistency with mechs
/:cl:

### CODE REVIEW

**Comment by rye-rice** [comment]
hellll yeaa

**Comment by rye-rice** [comment]
god i love those pilot suits

**Comment by CapnMachaddish** [comment]
Absolutely fragnificent. Glad to see these come to fruition.

**Comment by triplezeta** [comment]
:+1:

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -889,3 +889,194 @@
 	strip_delay = 130
 	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
 	actions_types = list()
+
+///////////////Shiptest Additions////////////////////
+
+//Softsuit helmet light framework
+/obj/item/clothing/head/helmet/space/light/
```
Comment (MarkSuckerberg) [inline code/modules/clothing/spacesuits/hardsuit.dm]:
```suggestion
/obj/item/clothing/head/helmet/space/light
```

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -889,3 +889,194 @@
...
+	light_system = MOVABLE_LIGHT_DIRECTIONAL
+	light_range = 4
+	light_power = 1
+	light_on = FALSE
+	on = FALSE
+
+/obj/item/clothing/head/helmet/space/light/ComponentInitialize()
+	. = ..()
+	AddElement(/datum/element/update_icon_updates_onmob)
+
```
Comment (MarkSuckerberg) [inline code/modules/clothing/spacesuits/hardsuit.dm]:
```suggestion
```
Not needed, already added on the parent's component initialize

**Review by MarkSuckerberg — CHANGES_REQUESTED** [review]
Last touches

**Pull request merged**